src/amo/components/Categories.scss

```diff
@@ -0,0 +1,21 @@
+.Categories-list {
+  margin: 0;
+  padding: 0;
+}
+
+.Categories-listItem {
+  background: url('../img/icons/arrow.svg') 100% 50% no-repeat;
+  border-bottom: 2px solid #000;
+  display: block;
+  list-style: none;
+  margin: 0 auto;
+  padding: 0;
+  width: 95%;
+
+  a {
```

Comment:
No your way is totally better; I've been trying for that too and just missed this one. It means less nesting which is nice.